walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VCVerifiedIdSerializer.kt

```diff
@@ -3,15 +3,25 @@
 package com.microsoft.walletlibrary.verifiedid
 
 import com.microsoft.walletlibrary.did.sdk.credential.service.models.contracts.VerifiableCredentialContract
+import kotlinx.serialization.json.Json
+
+object VCVerifiedIdSerializer : VerifiedIdSerializer<String> {
```

Comment:
Will this be internal at some point? How does developer know whether Verified ID is a VCVerifiedID or a issuance v2 verified ID if not?

Reply:
Removed this class and reused existing serializer returning String. Serialize method that returns a VC from Verified ID considers both VCVerifiedID and Issuance V2 Verified ID and returns raw VC appropriately. But serializer method only creates a VCVerifiedID. It will be removed when VC doesn't have to converted anymore. It is a specific scenario which needs conversion and there is no way to differentiate if VCVerified ID or Issuance V2 Verified ID needs to be created from VC now. A VCVerified ID is created always.